presentation/tests.py

```diff
@@ -1,3 +1,58 @@
+from django.contrib.auth.models import AnonymousUser
+from django.core.urlresolvers import reverse
 from django.test import TestCase
+from django.test import RequestFactory
+
+from presentation.models import Presentation
+from presentation.views import PresentationList
+from warp.users.models import User
+
+
+class PresentationListTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.test_user = self.create_test_user()
+        self.presentation = Presentation(subject="subject",
```

Comment:
Sometimes.. random is dangerous on test case. cause when test is broken, we can't check what is wrong.